en,zh: release notes for 1.6.0-beta.1 (#2519)
pingcap/docs-tidb-operator: en/tidb-scheduler.md

TiDB Scheduler is a TiDB implementation of [Kubernetes scheduler extender](https://github.com/kubernetes/design-proposals-archive/blob/main/scheduling/scheduler_extender.md). TiDB Scheduler is used to add new scheduling rules to Kubernetes. This document introduces these new scheduling rules and how TiDB Scheduler works.

> **Note:**
>

> > TiDB Operator v1.6 and later versions no longer recommend deploying TiDB Scheduler.

Reviewer: ```suggestion
> Starting from TiDB Operator v1.6, it is not recommended to deploy TiDB Scheduler.
```
